src/Symfony/Component/Routing/Loader/Configurator/CollectionConfigurator.php

```diff
@@ -114,4 +114,12 @@ final public function host(string|array $host): static
 
         return $this;
     }
+
+    /**
+     * Create a new route with the collection's settings.
```

Comment:
Let's add a comment saying this method is actually used since it overrides the one from LocalizedRouteTrait

Reply:
Added a comment with `@see` pointing to the trait::method